Neos.ContentRepository.BehavioralTests/Classes/Command/PerformanceMeasurementService.php

```diff
@@ -154,9 +153,10 @@ private function createHierarchy(
 
     public function forkContentStream(): void
     {
-        $this->contentRepository->handle(ForkContentStream::create(
-            ContentStreamId::create(),
-            $this->contentStreamId,
-        ));
+        throw new \BadMethodCallException('not implemented');
+        // $this->contentRepository->handle(ForkContentStream::create(
+        //     ContentStreamId::create(),
+        //     $this->contentStreamId,
+        // ));
```

Comment:
as discussed with @kitsunet we should probably remove the whole service for now (in a separate PR so that we can revert once the lowlevel changes are done)
Christian is working on a different approach for benchmarks and will steal some ideas from this class